model/pipeline/src/main/proto/beam_runner_api.proto

```diff
@@ -32,6 +32,20 @@ option java_outer_classname = "RunnerApi";
 import "google/protobuf/any.proto";
 import "google/protobuf/descriptor.proto";
 
+message BeamConstants {
+  enum Constants {
+    // The maximum timestamp for the global window in milliseconds since Jan 1, 1970.
+    // Triggers use maxTimestamp to set timers' timestamp. Timers fires when
+    // the watermark passes their timestamps. So, the timestamp needs to be
+    // smaller than the TIMESTAMP_MAX_MILLIS.
```

Comment:
Should we also add TIMESTAMP_MAX_MILLIS as a constant here if they are different?

Reply:
Might as well add it here too while we're at it.